src/layers/transform/concatenate.cu

```diff
@@ -470,13 +470,13 @@ void bp_compute_impl(
 }
 
 // Explicit instantiation
-#define PROTO(T)                                        \
-  template class concatenate_layer<                     \
-    T, data_layout::DATA_PARALLEL, El::Device::GPU>;    \
-  template class concatenate_layer<                     \
-    T, data_layout::MODEL_PARALLEL, El::Device::GPU>
+#define PROTO(T)                                                        \
+  template class concatenate_layer<                                     \
+    T, data_layout::DATA_PARALLEL, El::Device::GPU>;                    \
+  template class concatenate_layer<                                     \
+    T, data_layout::MODEL_PARALLEL, El::Device::GPU>;
```

Comment:
pretty sure you don't need the last semi-colon

Reply:
Correct, also an artifact of other changes.